dvc/repo/add.py

```diff
@@ -194,6 +183,8 @@ def _create_stages(
         )
         if stage:
```

Comment:
Desc is worth taking into account when checking if the stage is cached. Right now it ignores it :slightly_frowning_face: